Clicker/Extensions/String+Shared.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  String+Shared.swift
+//  Clicker
+//
+//  Created by Kevin Chan on 9/10/18.
+//  Copyright © 2018 CornellAppDev. All rights reserved.
+//
+
+extension String {
+    func height(withConstrainedWidth width: CGFloat, font: UIFont) -> CGFloat {
```

Comment:
you gonna give credit to that one stack overflow post? :P

Reply:
LOL. I also took the `UIScrollViewDelegate` code from `CardController+Extension` from another stack overflow post :P